pysrc/bytewax/bigquery/bigquery_output.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from google.cloud import bigquery
+
+from bytewax.outputs import ManualOutputConfig
+
+def output_builder(table_ref):
+    """Constructs an output function to write to BigQuery"""
+    client = bigquery.Client()
+    table = client.get_table(table_ref)
+
+    def write_rows(rows):
+        errors = client.insert_rows_json(table, rows)  # Make an API request.
+        if errors != []:
+            logging.info("Errors while inserting rows: {}".format(errors))
```

Comment:
Is this what we want?

If a user passes each row individually, the row will an error will fail to insert. If the user passes an array of rows, the entire batch will fail to insert. 

Should they be able to choose to crash or log/where to log?

Reply:
I don't think it really matters at this point. I think I'd default to throwing an exception, but logging is also fine.

FWIW, I think the initial reason I had the capture operator emit downstream what comes in is just since we don't support non-linear dataflows yet, so it's hard to test stuff in-line. And longer term, I think a better way to handle this is to make the `capture` operator emit some sort of "Ok / Err" downstream and then the designer can be as fancy as they want dealing with errors. Then we can retrofit all of these to pass a relevant error object from the client lib API; maybe that's catching exceptions and wrapping them and sending them downstream as objects or maybe it's using the return value, but it'll depend on the API.